Technical-Steering-Committee/principles.md

```diff
@@ -0,0 +1,47 @@
+# Principles
+
+The aim of this document is to collect important values from the Zowe community and to help us identify the right direction for the project. When we state we will do something it means to the best of our knowledge and none of the statements should be perceived as a valid reason for a legal action against Zowe. 
+
+
+Zowe is a project that aims to bring modern interfaces - API, command line, and visual – to z/OS via an open source community. In case of disagreement within the Zowe community, the principles outlined here need to be taken into account when mediating.
+
+
+## Transparent and open implementation, processes and deliveries
+**Principle:** Open first
+
+We will plan and develop transparently and in open. We will build an inclusive environment where anyone willing can easily find a way to participate and contribute. 
+
+## Zowe does not compromise on mainframe security standards
+**Principle:** Security is paramount
+
+We commit to secure development practices. We'll follow/comply/adhere to/obey with /conform to cybersecurity regulations requirements. We will strive to never introduce a feature or change that circumvents or allows circumvention of z/OS security infrastructure. We will regularly/continuously perform vulnerability testing of the Zowe project source code and any used third party libraries. If vulnerabilities are found they will be resolved with fix or workaround will be provided.
```

Comment:
I think `regulations requirements` should either just be `regulations` or `regulatory requirements`

Reply:
regulations makes sense